src/main/java/com/arcbees/analytics/client/ClientAnalytics.java

```diff
@@ -22,31 +22,50 @@
 import com.arcbees.analytics.shared.AnalyticsImpl;
 import com.arcbees.analytics.shared.AnalyticsPlugin;
 import com.arcbees.analytics.shared.HitType;
+import com.arcbees.analytics.shared.ProtocolTranslator;
 import com.arcbees.analytics.shared.options.AnalyticsOptions;
 import com.arcbees.analytics.shared.options.CreateOptions;
 import com.arcbees.analytics.shared.options.GeneralOptions;
 import com.arcbees.analytics.shared.options.TimingOptions;
+import com.google.gwt.core.client.Callback;
 import com.google.gwt.core.client.Duration;
 import com.google.gwt.core.client.JavaScriptObject;
+import com.google.gwt.core.client.JsArray;
+import com.google.gwt.core.client.JsArrayMixed;
+import com.google.gwt.core.client.ScriptInjector;
+import com.google.gwt.http.client.Request;
+import com.google.gwt.http.client.RequestBuilder;
+import com.google.gwt.http.client.RequestCallback;
+import com.google.gwt.http.client.RequestException;
+import com.google.gwt.http.client.Response;
+import com.google.gwt.http.client.URL;
 import com.google.gwt.json.client.JSONArray;
 import com.google.gwt.json.client.JSONObject;
+import com.google.gwt.json.client.JSONParser;
 import com.google.gwt.json.client.JSONString;
 import com.google.gwt.json.client.JSONValue;
 import com.google.inject.Inject;
-import com.google.inject.name.Named;
 
 public class ClientAnalytics extends AnalyticsImpl {
     private final Map<String, Double> timingEvents = new HashMap<>();
-
-    @Inject
-    ClientAnalytics(@Named("gaAccount") final String userAccount,
-            @Named("uaAutoCreate") final boolean autoCreate) {
+    private final String fallbackPath;
+    
+     @Inject
+    ClientAnalytics(@GaAccount final String userAccount,
```

Comment:
And we usually don't use final unless it's needed, but it seems everything is final in this project..